web/src/app/administration/components/classification-details/classification-details.component.html

```diff
@@ -31,7 +31,7 @@ <h4 class="panel-header">{{classification.name}}&nbsp; [{{classification.type}}]
 
       <!-- DETAILED FIELDS -->
       <div class="panel-body" style="padding: 0">
-        <form #ClassificationForm="ngForm">
+        <ng-form #ClassificationForm="ngForm">
```

Comment:
What is the advantage of the ng-form tag?

Reply:
ng-form is the new convention of writing angular form https://angular.io/api/forms/NgForm#selectors